### PR TITLE
linux: windows: set evictionStrategy explicitely

### PR DESCRIPTION
**Pull request by fromanirh**
Bug-Url: https://bugzilla.redhat.com/1714000
Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ branches:
   - /^v[0-9]/
 env:
   global:
-  - KUBEVIRT_VER=v0.15.0-alpha.0
+  - KUBEVIRT_VER=v0.17.0
```
Comment (ksimon1) [inline .travis.yml]: PR is already created and waiting for merge https://github.com/kubevirt/common-templates/pull/65

**Comment by MarSik** [comment]
I do not think we want to merge this. We decided in the past that upstream should not enable live migration by default.

**Comment by MarSik** [comment]
Also the root cause was elsewhere - https://bugzilla.redhat.com/show_bug.cgi?id=1711457

**Comment by fromanirh** [comment]
Thanks @MarSik . I missed the past decision, and I thought we needed *both* the validator and the common templates fix. But looks like the validator fix could be sufficient, so I'm closing this PR.

**Pull request closed without merging**